dbt_subprojects/tokens/models/transfers_and_balances/apechain/tokens_apechain_transfers.sql

```diff
@@ -0,0 +1,26 @@
+{{config(
+     schema = 'tokens_apechain'
+     , alias = 'transfers'
+     , partition_by = ['block_month']
+     , materialized = 'incremental'
+     , file_format = 'delta'
+     , incremental_strategy = 'merge'
+     , unique_key = ['block_date','unique_key']
+     , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+     , post_hook='{{ expose_spells(blockchains = \'["apechain"]\',
+                                spell_type = "sector",
+                                spell_name = "tokens",
+                                contributors = \'["hosuke"]\') }}'
+ )
+}}
+
+{{
+     transfers_enrich(
+         base_transfers = ref('tokens_apechain_base_transfers')
+         , tokens_erc20_model = source('tokens', 'erc20')
+         , prices_model = source('prices', 'usd')
+         , evms_info_model = source('evms','info')
```

Comment:
these transfers models will be hard to test in CI, if we include in this PR for each chain, as `evms.info` is in different subproject and therefore won't get applied here. i assume this is why we saw 0 rows returned when looking?

Reply:
maybe we end up doing:
- PR for metadata
- PR for transfers / net transfers (tbd on the net transfers side, more to come soon)
- PR for DEXs

Reply:
I will remove transfers from this PR first.